### ISSUE TITLE
Clair failed to start due to error while running migrations in a fresh installation.

### ISSUE THREAD

**Issue by ielshal** [goharbor/harbor#13134]
when starting harbor clair service i am getting below error,

```
{"Event":"running database migrations","Level":"info","Location":"pgsql.go:216","Time":"2020-09-22 12:42:07.098838"}
{"Event":"pgsql: an error occured while running migrations: pq: null value in column \"dirty\" violates not-null constraint","Level":"fatal","Location":"main.go:97","Time":"2020-09-22 12:42:07.102389"}
```

Tested this with harbor `2.0.0` and `2.0.2` clean install .. Any idea what could be issue here .. interesting is that when i start clair service before core it works fine but core service fail to start with same error .. so what could be missing here or do we need to have a separate table for migration to each component ?

Also i see similar error was reported here [173](https://github.com/goharbor/harbor-cluster-operator/pull/173)

**Comment by bitsf** [comment]
you can delete folder /data which specified in harbor.yml first, then refresh install again.
and use v2.0.3

**Comment by ielshal** [comment]
Unfortunately this did not fix the issue . i tried this multiple times and even in different hosts but no luck .. any idea what could be reason of this issue is it a known bug/issue which should be fixed in 2.0.3 ?

**Comment by ielshal** [comment]
This is fixed now after retrying with totally fresh installation .. not sure what was different .. but all fine now and this issue can be closed
